Jenkinsfile

```diff
@@ -40,21 +40,21 @@ pipeline {
           }
           steps {
             script {
-              env.DOCKER_TAG = 'openpaas-branch-master'
+              env.DOCKER_TAG = 'branch-master'
               if (env.TAG_NAME) {
                 env.DOCKER_TAG = env.TAG_NAME
               }
 
               echo "Docker tag: ${env.DOCKER_TAG}"
 
               // Temporary retag image names
-              sh "docker tag linagora/openpaas-james-memory linagora/james-memory:${env.DOCKER_TAG}"
-              sh "docker tag linagora/openpaas-james-distributed linagora/james-rabbitmq-project:${env.DOCKER_TAG}"
-              sh "docker tag linagora/openpaas-james-distributed-es6-backport linagora/james-rabbitmq-project:${env.DOCKER_TAG}-esv6"
+              sh "docker tag linagora/openpaas-james-memory linagora/tmail-backend:memory-${env.DOCKER_TAG}"
```

Comment:
Can we modify directly the JIB set up to get the right images there too?

Deployment tests will likely be affected too.

Reply:
I could look at it but I am not sure... as all our images on our repo have the same name with a different tag, I'm not sure how messy that would be on JIB side.

Reply:
Or do you mean the name change in the jib like :
`linagora/openpaas-james-memory` -> `linagora/tmail-backend-memory`

Reply:
Yes